docs/content/docs/patterns/observability/tracing.md

```diff
@@ -9,22 +9,42 @@ aliases: ["tracing.md"]
 Open Service Mesh (OSM) allows optional deployment of Jaeger for tracing. Similarly, tracing can be enabled and customized during installation (`tracing` section in `values.yaml`) or at runtime by editing the `osm-config` ConfigMap. Tracing can be enabled, disabled and configured at any time to support BYO scenarios.
 
 ## Jaeger
-[Jaeger](https://www.jaegertracing.io/) is an open source tracing system used for monitoring and troubleshooting distributed systems. It can be deployed with OSM as a new instance or you may bring your own instance.
+[Jaeger](https://www.jaegertracing.io/) is an open source distributed tracing system used for monitoring and troubleshooting distributed systems. It allows you to get fine-grained metrics and distributed tracing information across your setup so that you can observe which microservices are communicating, where requests are going, and how long they are taking. You can use it to inspect for specific requests and responses to see how and when they happen.
+
+
+When deployed with tracing enabled, Jaeger is capable of receiving spans from Envoys in the mesh that can then be viewed and queried via port-forwarding on Jaeger's UI.
+
+
+It can be deployed with OSM as a new instance or you may bring your own instance.
 
 ### Automatic bring up
 By default, Jaeger deployment and tracing as a whole is disabled.
 
-Jaeger can be automatically deployed by using the `--deploy-jaeger` OSM CLI flag at install time or by toggling the `deployJaeger` value in `values.yaml`.
+A Jaeger instance can be automatically deployed by using the `--deploy-jaeger` OSM CLI flag at install time. This will bring up a Jaeger pod in the mesh namespace. To configure all participating Envoys to send spans to it, tracing must be enabled as well.
```

Comment:
```suggestion
A Jaeger instance can be automatically deployed by using the `--deploy-jaeger` OSM CLI flag at install time. This will bring up a Jaeger pod in the mesh namespace.
Additionally, OSM has to be instructed to enable tracing on the proxies; this is done via the `tracing` section on the configmap.
The following command will both deploy Jaeger and enable tracing during installation:
```